curl-static-musl/plan.sh

```diff
@@ -0,0 +1,53 @@
+
+source ../curl/plan.sh
+
+pkg_name=curl-static-musl
+pkg_distname=curl
```

Comment:
I think there needs to be a `pkg_description` in this file, because the linter that checks to see if all required variables are present doesn't follow the `source` statements.

You can [run pre-commit] yourself before pushing to avoid getting dinged by Travis.